SMLHelper/Crafting/RecipeData.cs

```diff
@@ -5,8 +5,8 @@
     /// <summary>
     /// A class that fully describes a recipe for a <see cref="TechType"/> identified item.
     /// </summary>
-    /// <seealso cref="ITechData" />
-    public class TechData : ITechData
+    /// <seealso cref="TechData" />
+    public class RecipeData
```

Comment:
**TODO** Separate SMLHelper's `TechData` and the new `RecipeData` into two separate files.  
These will be wrapped entirely by the preprocessor directives. 

```CS
#if SUBNAUTICA
public class TechData : ITechData
{
....
}
#endif
```
```CS
#if BELOWZERO
public class RecipeData
{
....
}
#endif
```